src/App.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import './App.css';
+import {
+  BrowserRouter as Router,
+  Routes,
+  Route,
+  Navigate,
+} from 'react-router-dom';
+import Sidebar from './components/sidebar/Sidebar';
+import MainPage from './components/main_page/MainPage';
+import LandingPage from './components/landing_page';
+import Login from './components/auth/login';
+import Signup from './components/auth/signup';
+import ReservationsPage from './components/reservations';
+import NewReservation from './components/newReservation';
+import CourseDetail from './components/CourseDetail';
+import CourseForm from './components/course/courseForm';
+import DeleteCoursePage from './components/DeleteCoursePage';
+import Logout from './components/logout';
+
+function App() {
```

Comment:
- [ ]  Please, kindly refactor this and all other functions in your project to ES6 arrow functions, so your code complies with the latest standards.

Refactoring is easy:

```javascript
const App = () => {
  // ...
}
```

## Explanation

The `function` keyword has been soft deprecated in almost all the software industry in favor of ES6 arrow functions. This means, almost nobody uses it anymore, so much that it's considered a non-best practice.

You can do a search in Vscode for the "function" word, and correct as needed.  Shouldn't take you more than 5-10 min so don't worry (I found 15):

![image](https://github.com/ab-noori/EduPlanner-Frontend/assets/57421823/bb9b90d4-d8f8-4204-b286-2fd057ea276c) 

That way, your code will meet the latest standards. 💪